components/components_CSS/Button/button.css

```diff
@@ -0,0 +1,103 @@
+@import url("/root.css");
+
+button{
+    border: none;
+}
+
+.btn{
+    
+    padding: 0.8rem 2.5rem;
+    color: white;
+    border-radius:  8px; ;
+    font-family: Georgia, 'Times New Roman', Times, serif;
+    font-size: 1.2rem;
+    cursor: pointer;
+    border: none;
+}
+
+/* ------------------------------button hover */
+button:hover{
+    box-shadow:5px 12px 18px rgb(180, 175, 175);
+}
+
+/* ------------------------------solid button styling */
+.btn-primary{
+    background-color:blue ;
+}
+
+.btn-secondary{
+    background-color: grey;
+}
+
+.btn-danger{
+    background-color: red;
+}
+
+.btn-disabled{
+    background-color: rgb(192, 184, 184);
+}
+
+/* ------------------------------outlined button styling */
+.btn-primary-outl{
+    color:blue ;
+    border: 2px solid blue;
+}
+.btn-primary-outl:hover{
+    background-color:blue ;
+    color: white;
+}
+
+.btn-secondary-outl{
+    color: grey;
+    border: 2px solid grey;
+}
+.btn-secondary-outl:hover{
+    background-color: grey;
+    color: white;
+}
+
+.btn-danger-outl{
+    color: red;
+    border: 2px solid red;
+}
+.btn-danger-outl:hover{
+    background-color: red;
+    color: white;
+}
+
+.btn-disabled-outl{
+    color: rgb(192, 184, 184);
+    border: 2px solid rgb(192, 184, 184);
+}
+.btn-disabled-outl:hover{
+    background-color: rgb(192, 184, 184);
+    color: white;
+}
+
+/* -----------------------------------icon button */
+.btn-icon{
+    color: grey;
+    cursor: pointer;
+}
+.btn-icon:active{
+    color: red;
+}
+
+/* --------------------------------float button */
+.float{
+    height: 3rem;
+    width: 3rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(14, 177, 241);
+    border-radius: 2rem;
+    transition: transform 0.2s;
+    color: white;
+    cursor: pointer;
+}
+
+.float:hover{
+    height: 3.5rem;
```

Comment:
use transform property instead on increasing font size.

Reply:
Thank you, changed to tranform now